src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -98,6 +98,8 @@
     @JsonProperty("private")
     private boolean _private;
 
+    private GHVisibility visibility;
+
```

Comment:
We were just having a discussion on another thread as to how to handle enums like this.  We settled on a storing them as `String` and converting when requested to avoid throwing exceptions if new values are ever added. 

See https://github.com/hub4j/github-api/pull/1064/commits/b8f00bc699c7de3667b236899fff6ec87d7038a8